.gitignore

```diff
@@ -4,5 +4,5 @@ gemfiles/*.lock
 Gemfile.lock
 pkg/*
 tmp/*
-*.sqlite
+*.sqlite*
```

Comment:
This would be good to be a separate PR is it related to: https://github.com/alphagov/gds-sso/pull/299 ?

Reply:
Not sure if it's related, it's more related to the combustion gem, when you run rspec, combustion creates couple of sqlite instances and appends random chars at the end `...sqlite-asd`, so just adding it in the gitignore not to accidentally commit them